### PR TITLE
Added Continuous Variable Quantum Classifiers: MNIST community demo

### PR DESCRIPTION
**Pull request by CatalinaAlbornoz**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] Ensure that your tutorial executes correctly, and conforms to the
      guidelines specified in the [README](../README.md).

- [ ] Add a thumbnail link to your tutorial in `beginner.rst`, or if a
      QML implementation, in `implementations.rst`.

- [ ] All QML tutorials conform to
      [PEP8 standards](https://www.python.org/dev/peps/pep-0008/).
      To auto format files, simply `pip install black`, and then
      run `black -l 100 path/to/file.py`.

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Title:**

**Summary:**

**Relevant references:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: demos_community.yaml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+- title: Continuous Variable Quantum Classifiers -  MNIST
+  author: Sophie Choe
+  date: 04/06/2022
+  code: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers
+
+  paper: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers/blob/main/MNIST_QNN.pdf
+  color: heavy-rain-gradient
+  description: |
+    MNIST dataset classifiers using different number of qumodes: classical and CV quantum hybrid networks.
+    Author affiliation: Portland State University, Electrical and Computer Engineering.
```
Comment (CatalinaAlbornoz) [inline demos_community.yaml:10]:
```suggestion
    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in ["Continuous variable quantum neural networks"](https://arxiv.org/pdf/1806.06871v1.pdf). The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.
    Author affiliation: Portland State University, Electrical and Computer Engineering.

```

File: demos_community.yaml [hunk 2]
```diff
@@ -1,3 +1,15 @@
+- title: Continuous Variable Quantum Classifiers -  MNIST
+  author: Sophie Choe
+  date: 04/06/2022
+  code: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers
+
+  paper: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers/blob/main/MNIST_QNN.pdf
+  color: heavy-rain-gradient
+  description: |
+    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in ["Continuous variable quantum neural networks"](https://arxiv.org/pdf/1806.06871v1.pdf). The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.
```
Comment (CatalinaAlbornoz) [inline demos_community.yaml:9]:
```suggestion
    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in "Continuous variable quantum neural networks". The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.

```

Comment (CatalinaAlbornoz) [inline demos_community.yaml:9]:
```suggestion
    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in the paper "Continuous variable quantum neural networks" by Killoran et al. The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.

```

**Review by alvaro-at-xanadu — COMMENTED** [review]
The card looks good and it is an interesting demo. The only issue I have found is that the 4_qumode_classifier notebook does not run because the `init_layer` and `layer` functions are called in the quantum device, but they're not defined previously. Therefore, the classifier does not run (only in the 4 qumode case).

File: demos_community.yaml [hunk 3]
```diff
@@ -1,3 +1,16 @@
+- title: Continuous Variable Quantum Classifiers -  MNIST
+  author: Sophie Choe
+  date: 04/06/2022
+  code: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers
+
+  paper: https://github.com/sophchoe/Continous-Variable-Quantum-MNIST-Classifiers/blob/main/MNIST_QNN.pdf
+  color: heavy-rain-gradient
+  description: |
+    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in the paper "Continuous variable quantum neural networks" by Killoran et al. The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.
```
Comment (CatalinaAlbornoz) [inline demos_community.yaml:9]:
```suggestion
    We built 8 MNIST dataset classifiers using 2-8 qumodes. This family of MNIST classifiers are classical-quantum hybrid circuits using Keras and PennyLane. The quantum circuit is composed of a data encoding circuit and a quantum neural network circuit as proposed in the paper "Continuous variable quantum neural networks" by Killoran et al. The PennyLane-TensorFlow interface converts the quantum circuit into a Keras layer, and the whole network is treated as a Keras network, to which Keras' built-in loss function and optimizer can be applied for parameter updates. Categorical cross-entropy is used as the loss function and Stochastic Gradient Descent is used for the optimizer.

```

**Review by josh146 — APPROVED** [review]

**Pull request merged**